### PR TITLE
Add an example for integration tests using secretjs

### PR DESCRIPTION
**Pull request by liorbond**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Secret Contract Integration Tests
+
+This is an example of how to write an integration test using our "secretjs" module.
```
Comment (assafmo) [inline tests/README.md]:
```suggestion
This is an example of how to write an integration test using our "secretjs" module.
```
```suggestion
This is an example of how to write integration tests in Typescript using secret.js.
```

**Comment by assafmo** [comment]
Can you add a package.json?

**Comment by assafmo** [comment]
Also, maybe a readme with how to debug the ts tests with ts-nose and js debug console in vscode?

File: tests/README.md (hunk 1 above)
Comment (liorbond) [reply tests/README.md]: done

**Comment by reuvenpo** [comment]
Cool stuff!
The package.json should go in the main directory though.
Also, can you add the function i wrote [here](https://github.com/scrtlabs/LegenDAO/commit/62385d12a1f01d2910f81717ccf8cd764240162b#diff-aae216853526bb5eb76ffeff66cc7f56184296adde901fd9d48c08e925723a8aR37) ([usage](https://github.com/scrtlabs/LegenDAO/commit/62385d12a1f01d2910f81717ccf8cd764240162b#diff-66c84fc6394a745be0dd169a0aef1cde602eadfd19a75f613d7e6ef9e2b1ecf0L129)) instead of the loop near the beginning of the test code?

**Comment by assafmo** [comment]
CI too please.

**Review by assafmo — APPROVED** [review]

**Pull request merged**